packages/react/src/Image/Image.test.tsx

```diff
@@ -0,0 +1,141 @@
+import React from 'react'
+import {render, cleanup} from '@testing-library/react'
+import '@testing-library/jest-dom'
+import {axe, toHaveNoViolations} from 'jest-axe'
+
+import {Image} from './'
+
+expect.extend(toHaveNoViolations)
+
+describe('Image', () => {
+  afterEach(cleanup)
+
+  it('no a11y violations', async () => {
+    const {container} = render(<Image src="https://source.unsplash.com/random" alt="" aspectRatio={undefined} />)
```

Comment:
Noticing `aspectRatio={undefined}` on quite a few of these tests, could we remove them as it's not a required prop.

also instead of `""` for the alt text, could we put something like 'mock alt text' please?

```suggestion
    const {container} = render(<Image src="https://source.unsplash.com/random" alt="mock alt text" />)
```